README.md

```diff
@@ -17,13 +17,10 @@ publisher that pushes media atoms into a Kinesis stream.
    [here](https://github.com/guardian/dev-nginx#nginx-dev-setup) to set up
    nginx locally.
 0. The nginx bucket size should be `128`, up from the default of `64`. Edit `nginx.conf` and modify the `http` section to contain `server_names_hash_bucket_size 128;`
-0. Run `./scripts/setup-dev-conf` to generate an `application.conf` file. You need to pass to this script the name of the stack that you want to use
-   (usually this is the default `media-atom-maker-dev` stack),
-   aws profile for panda (e.g. `media-service`) and the domain for the app
-   (`local.dev-gutools.co.uk`)
-0. If you need your own dev-stack, you can create it by using the
-   `media-atom-maker-dev.yml` template and by generating the `application.conf`
-   file from the sample `reference.conf` file in the cloudformation folder
+0. You'll need to get the private configuration information from the S3 bucket (atom-maker-conf) and put it in your /etc/gu folder. This will be referenced in `application.conf`
+ on start up.
+`aws s3 cp s3://atom-maker-conf/DEV/atom-maker.private.conf /etc/gu/atom-maker.private.conf --profile media-service`
```

Comment:
would `media-atom-maker.private.conf` be better, as there's less chance of clashing with any config from `atom workshop`.